CMakeLists.txt

```diff
@@ -57,14 +57,20 @@ add_library(beman::any_view ALIAS beman.any_view)
 
 set_target_properties(
     beman.any_view
-    PROPERTIES VERIFY_INTERFACE_HEADER_SETS ON EXPORT_NAME any_view
+    PROPERTIES
+        # cmake-format: off
+        VERIFY_INTERFACE_HEADER_SETS ON
+        EXPORT_NAME any_view
```

Comment:
I am surprised, the linter is definitely trying to understand pairs, and listed `set_target_properties` explicitly as a site to detect pairs. Maybe two pairs is too little pairs?

See: https://github.com/BlankSpruce/gersemi?tab=readme-ov-file#style

Reply:
> gersemi will try to format the code in a way that respects set character limit for single line and only break line whenever necessary with one exception. The commands that have a group of parameters that aren't attached to any specific keyword (like set or list(APPEND)) will be broken into multiple lines when there are more than 4 arguments in that group.